src/client/renderer/r_state.c

```diff
@@ -868,9 +868,10 @@ void R_InitSupersample(void) {
 
 	r_state.supersample_image->width = r_context.render_width;
 	r_state.supersample_image->height = r_context.render_height;
+	r_state.supersample_image->layers = 0;
 	r_state.supersample_image->type = IT_NULL;
 
-	R_UploadImage(r_state.supersample_image, GL_RGBA, NULL);
+	R_UploadImage(r_state.supersample_image, r_hdr->integer ? GL_RGBA16 : GL_RGBA8, NULL);
```

Comment:
Should this be `r_hdr_enabled` .. err `r_using_hdr` ?

Reply:
Can be, it's not that much required here, but it's logical to change this.